.circleci/reference/HJJ_WW.lhe

```diff
@@ -1,6 +1,6 @@
 <LesHouchesEvents version="1.0">
 <!--
-Output from the JHUGenerator v7.5.0 described in arXiv:1001.3396 [hep-ph], arXiv:1208.4018 [hep-ph], arXiv:1309.4819 [hep-ph], arXiv:1606.03107 [hep-ph]
+Output from the JHUGenerator v7.5.1 described in arXiv:1001.3396 [hep-ph], arXiv:1208.4018 [hep-ph], arXiv:1309.4819 [hep-ph], arXiv:1606.03107 [hep-ph]
```

Comment:
For example over here, why do these lines even have to change? I don't think they should be part of the diffs between reference files and the file that is generated during CC checks.

Reply:
These lines are a part of the diffs

Reply:
Understood. Is there a way to not make them part of the diffs so that we don't have to replace these files every time JHUGen version changes?

Reply:
I fixed this so diff doesn't check the first few lines which include the version number

Reply:
Yes, I saw that. I think you still need to propagate the name changes to MELA; otherwise the checks would fail atm.